style.css

```
@@ -1122,6 +1122,11 @@ details summary:focus,
   background-color: var(--color-button-bg-hover);
 }
 
+#exportCodeButton.no-autosave {
+  outline: 2px dashed currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)